swagger.yaml

```diff
@@ -0,0 +1,107 @@
+
+
+---
+swagger: "2.0"
+info:
+  title: REST API
+  description: description in repository on github https://github.com/Raskolnikofff/StudyWebServerRestFullAPI
+  version: 0.1.9
+tags:
+- name: "student on course"
+  description: "Everything about your student"
+paths:
+  /: 
+    get:
+      summary: "open homepage"
+      tags:
+      - "student on course"
+      responses:
+        '200':
+           description: 'OK'
+  /students:
+    get:
+      summary: "find all students on course"
+      tags:
+      - "student on course"
+      responses:
+        '200':
+           description: 'OK'
+  /students/{id}:
+    get:
+      parameters:
+      - name: "id"
+        in: "path"
+        description: "ID of student to return"
+        required: true
+        type: "integer"
+        format: "int64"
+      summary: "Find students with ID"
+      tags:
+      - "student on course"
+      responses:
+        '200':
+          description: 'OK'
+    post:
```

Comment:
Semantic error at paths./students/{id}.post
Operations with parameters of "in: formData" must include "application/x-www-form-urlencoded" or "multipart/form-data" in their "consumes" property
Jump to line 44

Check with https://editor.swagger.io/